Store files in ~/.local/share/mcfly/ instead of ~/.mcfly/ to avoid polluting
HOME directory.

Upstream-Issue: https://github.com/cantino/mcfly/pull/124

--- a/src/settings.rs
+++ b/src/settings.rs
@@ -448,7 +448,9 @@
     pub fn storage_dir_path() -> PathBuf {
         home_dir()
             .unwrap_or_else(|| panic!("McFly error: Unable to access home directory"))
-            .join(PathBuf::from(".mcfly"))
+            .join(PathBuf::from(".local"))
+            .join(PathBuf::from("share"))
+            .join(PathBuf::from("mcfly"))
     }
 
     pub fn mcfly_db_path() -> PathBuf {
